migrations/V162__Update_MDM_To_MDM_POR.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TYPE OPERATIONAL_SOLUTION_KEY RENAME VALUE 'MDM' TO 'MDM-POR';

```

Comment:
```suggestion
ALTER TYPE OPERATIONAL_SOLUTION_KEY RENAME VALUE 'MDM' TO 'MDM_POR';
```
I think that `_` might be more consistent